tests/commontests.py

```diff
@@ -61,25 +61,26 @@ def test_include(self):
                            tarchecker=self.assertIncludeSubdirTar)
 
     def test_absolute_subdir(self):
-        (stdout, stderr, ret) = \
-            self.tar_scm_std_fail('--subdir', '/')
+        # ret = (stdout, stderr, return_code)
+        ret = self.tar_scm_std_fail('--subdir', '/')
```

Comment:
The idiomatic way to avoid this warning is prefixing them with `_`:

    (_stdout, _stderr, return_code) = \
        ...